### ISSUE TITLE
works in ruby 1.9.2, not 1.9.3

### ISSUE THREAD

**Issue by bwlang** [grimen/validatious-on-rails#40]
I think this project is abandoned... but in case others are still using it.

/Users/langhorst/.rvm/gems/ruby-1.9.3-p125@global/gems/bundler-1.1.3/lib/bundler/runtime.rb:74:in `require': cannot load such file -- validatious/on/rails (LoadError)
=> ruby-1.9.3-p125 [ x86_64 ]

works fine if i rvm use ruby 1.9.2

**Comment by grimen** [comment]
Yep I abandoned this 3 years ago I'm afraid. =/ There are a few newer gems that are maintained and, such as:

https://github.com/bcardarella/client_side_validations
https://github.com/overture8/green_light

Moving to them should be simpler I think.
